lt/client.go

```diff
@@ -626,6 +639,13 @@ func (u *user) Connect(stopCh chan struct{}, channelType model.ChannelType) erro
 	}
 
 	log.Printf("%s: disconnecting...", u.cfg.username)
+
+	defer func() {
+		if err := recover(); err != nil {
+			log.Printf("%s: caught a panic, probably closing an already closed channel. That's expected bc of the sim. Err: %v\n",
+				u.cfg.username, err)
+		}
+	}()
```

Comment:
Not a huge fan of `recover` unless it's unfeasible to avoid it. I think it may be time to deepen a bit the abstraction here and consider decoupling the highest level, load-test user state from the calls client which can become its own type/package.